app/pages/cif/project-revision/[projectRevision]/index.tsx

```diff
@@ -87,12 +226,162 @@ export function ProjectRevision({
 
   const taskList = <TaskList projectRevision={query.projectRevision} />;
 
+  const isOverviewEmpty =
+    Object.keys(query.projectRevision.projectFormChange.newFormData).length ===
+    0;
+
+  const areManagersEmpty =
+    query.projectRevision.projectManagerFormChangesByLabel.edges
+      .map(({ node }) => node.formChange?.newFormData.cifUserId)
+      .filter((manager) => manager).length === 0
+      ? true
+      : false;
+
+  // need to check for the existence of contactId because users can add blank contacts
+  const areContactsEmpty =
+    query.projectRevision.projectContactFormChanges.edges
+      .map(({ node }) => node.newFormData.contactId)
+      .filter((contact) => contact).length === 0
+      ? true
+      : false;
+
+  const areSecondaryContactsEmpty =
+    secondaryContacts
+      .map(({ node }) => node.newFormData.contactId)
+      .filter((contact) => contact).length === 0
+      ? true
+      : false;
+
   return (
     <DefaultLayout session={query.session} leftSideNav={taskList}>
       <div>
         <header>
           <h2>Review and Submit Project</h2>
         </header>
+        <h3>Project Overview</h3>
+        {isOverviewEmpty ? (
+          <p>
+            <em>Project overview not added</em>
+          </p>
+        ) : (
+          <FormBase
+            tagName={"dl"}
+            theme={readOnlyTheme}
+            schema={projectSchema as JSONSchema7}
+            uiSchema={createProjectUiSchema(
+              selectedOperator ? selectedOperator.node.tradeName : "",
+              selectedOperator ? selectedOperator.node.legalName : "",
+              selectedOperator ? selectedOperator.node.bcRegistryId : "",
+              rfpStream
+                ? `${rfpStream.node.fundingStreamByFundingStreamId.description} - ${rfpStream.node.year}`
+                : "",
+              projectStatus ? projectStatus.node.name : ""
+            )}
+            formData={query.projectRevision.projectFormChange.newFormData}
+            formContext={{
+              query,
+              form: query.projectRevision.projectFormChange.newFormData,
+            }}
+          />
+        )}
+
+        <h3>Project Managers</h3>
+        {areManagersEmpty ? (
+          <p>
+            <em>Project managers not added</em>
+          </p>
+        ) : (
+          query.projectRevision.projectManagerFormChangesByLabel.edges.map(
```

Comment:
Right now this component isn't rendered more than once, but that will change once the change reason input is added.
This is potentially a performance issue, as you will re-render the forms on every render (i.e. every character typed). A `useMemo` should be used so that you don't create new components on every render